sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/LocalModelFetcher.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Text;
+using System.Collections.Generic;
+using System.Threading;
+using System.Globalization;
+
+namespace Azure.Iot.ModelsRepository.Fetchers
+{
+    internal class LocalModelFetcher : IModelFetcher
+    {
+        private readonly bool _tryExpanded;
+
+        public LocalModelFetcher(ResolverClientOptions clientOptions)
+        {
+            _tryExpanded = clientOptions.DependencyResolution == DependencyResolutionOption.TryFromExpanded;
```

Comment:
Possibly null?

Reply:
clientOptions?.DependencyResolution

Reply:
DependencyResolution is an enum, not nullable.

Reply:
clientOptions could be null